source/concepts/protocols.rst

```diff
@@ -0,0 +1,310 @@
+discussions.
+
+Ledger channel
+--------------
+
+   - Formed between any two parties having sufficient funds in their on-chain
+     accounts.
+   - The funds are directly deposited into the smart contracts on the
+     blockchain.
+   - Allows any number of off-chain transactions between the participants.
+   - The channel must settled on the blockchain.
+
+Sub-channel
+-----------
+
+   - Formed between participants who have a ledger channel established between
+     them.
+   - Funds are locked on the ledger channel. Hence, no on-chain interaction
+     required to setup a sub-channel.
+   - Sub-Channel is settled,
+
+     - by unlocking the funds from the ledger channel, if both parties agree on
+       the final state of the channel through off-chain transactions; or
+     - by settling the parent channel on the blockchain, otherwise.
+
+       In this case, the parent channel and all other sub-channels opened using
+       the parent channel will also be settled on the blockhain.
+
+Virtual channel
+---------------
+
+   - Formed between two participants who do not have a ledger channel
+     established between them, but each of them has a ledger
+     channel established with a common intermediary.
+   - Funds are locked in the two ledger channels that the participants have
+     with the intermediary. Hence, no on-chain interaction required.
+   - Channel is settled,
+
+     - by unlocking the funds from the two ledger channel, if both parties
+       agree on the final state of the channel through off-chain transactions;
+       or
+     - by settling either one or both (depending on the scenario)
+       parent channels on the blockchain, otherwise.
+
+       In this case, the parent channels and all other sub-channels, virtual
+       channels opened using the parent channel will also be settled on the
+       blockhain.
+
+.. note:
+
+   From the above descriptions, it can be seen that sub-channels and virtual
+   channels require **zero on-chain** interactions under normal circumstances.
+   On-chain interactions are required only when the participants do not agree on
+   the state to be settled.
+
+In the next section, protocols for each of the four phases in
+the life cycle of a state channel are described.
+
+
+Phases of a state channel
+=========================
+
+Open phase
+----------
+
+In this phase, the channel is set up for off-chain transactions. Channel
+opening consists of two phases:
+
+1. Channel Proposal
+2. Funding
+
+The channel proposal protocol is same for all for three types of channels.
+However, the ``parent layer`` used for funding and the funding protocol are
+different:
+
+1. for ledger channels it is the blockchain,
+2. for sub-channels it is the ledger channel between the same participants, and
+3. for virtual channels the ``parent layer`` consists of the two ledger
+   channels between each of the channel participants and a common intermediary.
```

Comment:
does it make sense to add for this text as well an overview picture for better understanding?

Reply:
Yes, as we discussed in the call, we could add a picture here and move this to a separate page.